<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmRotate</class>
 <widget class="QWidget" name="CArmRotate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>696</width>
    <height>122</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>696</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>104</number>
   </property>
   <property name="rightMargin">
    <number>92</number>
   </property>
   <property name="bottomMargin">
    <number>35</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="rotateWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <widget class="QLabel" name="rotateValue">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>4</y>
        <width>124</width>
        <height>54</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>124</width>
        <height>54</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="TouchPanelSlider" name="rotateSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>71</y>
        <width>500</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TouchPanelSlider</class>
   <extends>QSlider</extends>
   <header>CArmTouchPanelSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
